src/methods/method_overloading/hard/q7/Distances.java

```diff
@@ -0,0 +1,41 @@
+package methods.method_overloading.hard.q7;
+
+/*
+Write a Java Program that calculates the distance between 2 points given by the user. The first set of points are
```

Comment:
Do you think it might be a good idea to give the students one of the two points in each set so we can add variation on the website?